deprecated/README.md

```diff
@@ -1 +1,11 @@
-TBD
\ No newline at end of file
+# Deprecated folder
+
+This folder contains all rules whose status has been set to deprecated.
+
+It is preferable not to use them any more
+
+the file [deprecated.csv](deprecated.csv) is a summary

```

Comment:
```suggestion
This folder contains all rules that have been marked as deprecated.

It is recommended to avoid using these rules, as they are no longer maintained or supported.

For a summary, refer to the file [deprecated.csv](./deprecated.csv)

```